frontend/app/components/Search/SearchBar/extract-ad-disclosure.ts

```diff
@@ -0,0 +1,39 @@
+import type { Hit } from "instantsearch.js";
+
+const STRAPI_BASE_URL =
+  process.env.NODE_ENV === "production"
+    ? "https://sublime-vitality-a5729e6407.strapiapp.com"
```

Comment:
Can this be provided via an env var instead so we don't need to have these hard coded?